system-addon/content-src/asrouter/components/ModalOverlay/ModalOverlay.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+export class ModalOverlay extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.onClick = this.onClick.bind(this);
+  }
+
+  componentWillMount() {
+    this.setState({active: true});
+  }
+
+  componentWillReceiveProps() {
+    this.setState({active: true});
+  }
+
+  onClick(e) {
```

Comment:
can you make this a little bit more descriptive like, `onDoneButtonClick` or something like that